doc/modules/ROOT/pages/migration-gds-1-to-gds-2/index.adoc

```diff
@@ -1,6 +1,6 @@
 [appendix]
 [[appendix-b]]
-= Migration from Graph Data Science library Version 1.x
+= Migrate from GDS v.1.x
```

Comment:
In general, do we prefer verbs here rather than nouns? I'm thinking about "Migrate" vs "Migration". I don't have a preference, but I'm curious if there's any underlying thinking?

Reply:
I think this title change is fine, except we've been advised by Marketing to always spell out GDS in its full form, so perhaps we retain that
```suggestion
= Migrate from Graph Data Science v.1.x
```